draft-uma-claims-sec-ext-v1_0.xml

```diff
@@ -239,11 +272,19 @@
           style="symbols">
           <t>Josh Mandel</t>
 
-          <t>Justin Richer</t>
-
           <t>Sarah Squire</t>
         </list></t>
     </section>
+
+    <section title="Security Considerations">
+      <t>An authorization server must not allow clients to use both the old
```

Comment:
Suggest rewording slightly to make this a rationale for the swap of endpoints: Something like "It is essential for an authorization server to handle only one kind of client. Therefore, the authorization server must..."